includes/amp-helper-functions.php

```diff
@@ -213,14 +213,12 @@ function() {
 					style.appendChild( document.createTextNode( 'html:not(#_) { margin-top: 0 !important; } #wpadminbar { display: none !important; }' ) );
```

Comment:
The style that hides the admin bar will get loaded once all the DOM elements are loaded, which means it will show the admin bar for brief movement (it will be noticeable on slow internet connections)

Can you please check if we can inject this style directly without JS, Remember the style needs to be injected only on onboarding preview, not the frontend version